src/libraries/Microsoft.Extensions.DependencyInjection/src/CallSiteJsonFormatter.cs

```diff
@@ -159,7 +159,7 @@ public void StartProperty(string name)
                 {
                     _firstItem = false;
                 }
-                Builder.AppendFormat("\"{0}\":", name);
+                Builder.Append($"\"{name}\":");
```

Comment:
@eerhardt, opinions on taking changes like this that will improve netcoreappcurrent but add cost to downlevel?

Reply:
> Thinking about our customers though, what is our recommendation for a "normal multi-targeted library"?

The same for any new APIs: decide whether the extra maintenance cost of maintaining two different code paths is worth the benefits.  In the case of new APIs, the alternate being weighed is not getting the benefit at all up-level.  In this case, the alternate being weighed is paying more downlevel.

Reply:
All 3 places being changed in DependencyInjection are in non-performance-critical paths. 2 in EventSource logging, 1 in building an exception message.

I guess just like the other libraries, I would revert these changes unless we had data backing up why we should make them.

Reply:
Oh, OK then, I will revert them in DI as well.